.bazelrc

```diff
@@ -24,7 +24,7 @@ build --stamp --workspace_status_command tools/workspace_status.sh
 build:race --@io_bazel_rules_go//go/config:race --@io_bazel_rules_go//go/config:pure=false
 test:race --@io_bazel_rules_go//go/config:race --@io_bazel_rules_go//go/config:pure=false
 
-build --@io_bazel_rules_go//go/config:pure
+build --@io_bazel_rules_go//go/config:pure=false
```

Comment:
When we test locally with bazel build to compile runsc binary, we found that if pure is not set as false, all packages that import cgo related packages will not be compiled. So we add pure=false in .bazelrc. Any suggestion on how to make more neat?

Reply:
Thanks for pointing out this! 
To avoid setting the whole runsc binary (even without using plugin stack) as non-pure, we add a "cgo-enable" build config to indicate whether cgo should be used.